### PR TITLE
Add 4 pages for troubleshooting and re-organize its sidebar

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/troubleshoot/troubleshooting-common-errors-and-solutions.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title: Common errors and solutions
+slug: /troubleshooting-common-errors-and-solutions
+---
+<head>
+  <link rel="canonical" href="https://docs.risingwave.com/docs/current/troubleshooting-common-errors-and-solutions/" />
+</head>
+
+This page summarizes some common issues encountered in various scenarios and provides corresponding solutions.
+
+## RisingWave is terminated with exit code `132`
```
Comment (lmatz) [inline docs/troubleshoot/troubleshooting-common-errors-and-solutions.md]: This one is nice, as the exit code is not controlled and explained by Risingwave itself

**Review by WanYixian — APPROVED** [review]
LGTM

**Comment by hengm3467** [comment]
@lmatz Could you please take a look? Thanks!

File: docs/troubleshoot/troubleshoot-oom.md [hunk 2]
```diff
@@ -24,7 +25,7 @@ This guide focuses on addressing OOM issues on the compute node. If you encounte
 
 ## OOM when creating materialized views
 
-If OOM happens during creating a new materialized view, it might be caused by the large amount of existing data in upstream systems like Kafka. In this case, before creating or recreating a materialized view, you can reduce the traffic by specifying the rate limit of each parallelism:
+If OOM happens during the creation of a new materialized view, it might be caused by the large amount of existing data in upstream systems like Kafka. In this case, before creating or recreating a materialized view, you can reduce the traffic by specifying the rate limit of each parallelism:
```
Comment (lmatz) [inline docs/troubleshoot/troubleshoot-oom.md]: > you can reduce the traffic by specifying the rate limit of each parallelism

or you can set `streaming_parallelism` to a smaller number.

**Review by lmatz — APPROVED** [review]
LGTM

**Review by CharlieSYH — APPROVED** [review]

File: docs/troubleshoot/troubleshoot-meta.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+id: troubleshoot-meta
+title: Meta failure
+slug: /troubleshoot-meta
+---
+
+## Overview
+
+If the meta fails to start or suddenly exits in the middle, currently only one issue has been found that will cause this problem: the meta node failed to stay alive while participating in the campaign.
```
Comment (hengm3467) [inline docs/troubleshoot/troubleshoot-meta.md]: @ShanlanLi  Please make sure to improve the wording with the help of the editor bot.

**Pull request merged**